### PR TITLE
Fix Known Issues

### PR DESCRIPTION
**Pull request by sascala**
## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-website#testing-your-updates-locally) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-website#contribution-workflow).

### CODE REVIEW

**Comment by sascala** [comment]
@karlkfi Does my change to the beta 1 release page address the problem?

File: src/releases/1.10-beta2/notes.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ The following updated data services packages are also in beta and are compatible
 # Breaking Changes
 
 - Upgrades not supported from 1.10.0 Beta 1 to 1.10.0 Beta 2.
-  Upgrades from 1.10.0 Beta 1 to 1.10.0 Beta 2 are _not supported_ in 1.10.0 Beta 2. This upgrade will be supported in a future 1.10 release.
+  Upgrades from 1.10.0 Beta 1 to 1.10.0 Beta 2 are _not supported_ in 1.10.0 Beta 2. This Upgrade will be supported in a future 1.10 release.
```
Comment (karlkfi) [inline src/releases/1.10-beta2/notes.md]: why capitalized?

**Comment by judithpatudith** [comment]
There isn't a link there yet... I was planning on adding the 1.10 section with all the releases once 1.10 goes GA, but I can definitely see the advantage of adding the section now so that everyone has access to all the betas.

**Comment by karlkfi** [comment]
> I was planning on adding the 1.10 section with all the releases once 1.10 goes GA

I would recommend having one once we have an RC at least. 
It was just occurring to me that the beta1 page had no incoming links now that there's a beta2 and the release page shows no indication of it... like it's not a real release or something...
I think as soon as we have 2 releases (e.g. beta1 & beta2) we should have a section for the new release. That way we can give the short list of new features and say "COMING SOON" or "BETA" in big text or something.

**Comment by judithpatudith** [comment]
👍  expect a PR soon :)

**Pull request merged**